pkg/email/help_send_feedback.go

```diff
@@ -32,13 +31,14 @@ func (c Client) SendFeedbackEmail(ctx context.Context, input SendFeedbackEmailIn
 	err := c.templates.helpSendFeedback.Execute(&b, input)
 
 	if err != nil {
-		return &apperrors.NotificationError{Err: err, DestinationType: apperrors.DestinationTypeEmail}
+		return err
 	}
 
-	err = c.sender.Send(ctx, []models.EmailAddress{c.config.EASIHelpEmail}, nil, subject, b.String())
-	if err != nil {
-		return &apperrors.NotificationError{Err: err, DestinationType: apperrors.DestinationTypeEmail}
-	}
-
-	return nil
+	return c.sender.Send(
+		ctx,
+		NewEmail().
+			WithToAddresses([]models.EmailAddress{c.config.EASIHelpEmail}).
+			WithSubject(subject).
+			WithBody(b.String()),
```

Comment:
Aside from the shorter function signature, what's the benefit here to passing in the email and having to call all these `With` methods versus passing the arguments? Is it that we get to avoid passing `nil` if we don't want to fill out a field?

Reply:
yeah, I wanted to avoid passing in `nil` in two consecutive args everywhere (for CC and BCC). BCC is only used in one place, and I don't believe I found an instance of CC being used.

If there are objects to the refactor, please let me know! Happy to discuss and certainly not married to any one way

Reply:
It's a little weird chaining the `NewEmail` helper with the other methods, but I think it's better than having 5 arguments and having to `nil` them out constantly. No objections.